doc/modules/changes/20201110_kchotalia

```diff
@@ -0,0 +1,5 @@
+New: Particles now supported for a spherical shell geometry with periodicity
+in polar angle direction for a 2D quarter shell (90 degree opening angle).
+<br>
+
+(Kiran Chotalia, Rene Gassmoeller, 2020/11/10)
```

Comment:
you misunderstood: please remove the empty line after the <br> and at the newline on the last line again.